### PR TITLE
scribus: Add an icon

### PR DESCRIPTION
**Pull request by ejpcmac**
###### Motivation for this change

Scribus has no desktop icon currently. I’ve checked in the source tarball, and there is nothing good here: only a low-res icon.

###### Things done

I’ve got a SVG file from [this page](https://wiki.scribus.net/canvas/Promotion_material) on the Scribus Wiki and edited it a bit to get a square file and center the circle. Since the icon is not present in the source tarball, I’ve put this resource in `nixpkgs`. I’m not sure if it is a good thing or not, let me know if you have a better idea on how to do this.

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

I have also checked the possibility to backport it to `release-18.09`: it would require one more action to create the `.desktop` file since Scribus 1.4.6 does not include one. I have already defined one for a wrapper in my [Nix channel](https://github.com/ejpcmac/jpc-nix/blob/master/pkgs/scribus-with-icon/default.nix#L6-L13) so I could port it easily to `release-18.09` if it is OK to proceed.

### CODE REVIEW

File: pkgs/applications/office/scribus/default.nix [hunk 1]
```diff
@@ -1,8 +1,9 @@
 { stdenv, fetchurl, pkgconfig, freetype, lcms, libtiff, libxml2
 , libart_lgpl, qt4, python2, cups, fontconfig, libjpeg
-, zlib, libpng, xorg, cairo, podofo, aspell, boost, cmake }:
+, zlib, libpng, xorg, cairo, podofo, aspell, boost, cmake, imagemagick }:
 
 let
+  icon = ./icon.svg;
```
Comment (matthewbauer) [inline pkgs/applications/office/scribus/default.nix]: Can we download this from somewhere? Ideally it would be included in the the src

Comment (ejpcmac) [reply pkgs/applications/office/scribus/default.nix]: No, the original SVG file is [here](http://linux.hanski.info/static/scribus_promotion/scribus-icon2.svg), but it is not square nor centered so I’ve edited it. If you know how we can script that on the SVG file or when converting it to PNG we are good, but else we need to use this custom file.

Comment (erictapen) [reply pkgs/applications/office/scribus/default.nix]: How about supplying icon.svg as a gist? This way it would not pollute the Nixpkgs source.

Comment (ejpcmac) [reply pkgs/applications/office/scribus/default.nix]: @erictapen Nice idea, I’ll do this.

Comment (ejpcmac) [reply pkgs/applications/office/scribus/default.nix]: Done. I can rebase and squash the commit if needed.

**Pull request merged**